k8s/scripts/minikube_setup.sh

```diff
@@ -14,14 +14,26 @@ set -e
 : ${KUBERNETES_VERSION:=v1.21.2}
 
 OS=$( uname -s | tr '[:upper:]' '[:lower:]' )
+ARCH=$( uname -m | sed 's/x86_64/amd64/')
```

Comment:
here as well

Reply:
Done - but note that this doesn't run in a container, so on Mac there's no need for an `aarch64` to `arm64` name conversion...but...for completeness and best support it's probably a good idea to have it!